infrastructure/terraform/per_account/mgmt/terraform.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  backend "s3" {
+    encrypt              = false
+    region               = "eu-west-2"
+    bucket               = "nhse-cpm--terraform-state-v1.0.0"
+    dynamodb_table       = "nhse-cpm--terraform-state-lock-v1.0.0"
+    key                  = "terraform-state-infrastructure-mgmt-dns"
```

Comment:
is dns the right suffix here?

Reply:
As discussed this is fine for now as it will cause issues with the bucket redeploying all our zones